File: Java Script - Introduction to Programming/2.2. Simple Calculations - Exam Problems/05. Daily Earnings.js
function dailyEarnings([...arg]) {
  let [workDayInMonth, moneyPerDay, usdPrice] = [...arg.map(parseFloat)]

  let mountCash = workDayInMonth * moneyPerDay
  let allMoney = mountCash * 12 + mountCash * 2.5
  let moneyCounter = allMoney / 100 * 25
  let moneyConvertToLeva = (allMoney - moneyCounter) * usdPrice
  let totalMoneyForDay = (moneyConvertToLeva / 365).toFixed(2)
  return totalMoneyForDay
}

//console.log(dailyEarnings(['21', '75.00', '1.59']))

File: Pyrhon - Introduction to Programming/2.2. Simple Calculations - Exam Problems/05. Daily Earnings.py
work_day_in_month, money_per_day, usd_price= [float(input()) for _ in range(3)]
mount_cash = work_day_in_month * money_per_day
all_money = mount_cash * 12 + mount_cash * 2.5
money_counter = all_money / 100 * 25
money_convert_to_leva = (all_money - money_counter) * usd_price
total_money_for_day = f'{(money_convert_to_leva / 365):.2f}'
print(total_money_for_day)


'''
21
75.00
1.59
#74.61
'''
